src/component/feature/FeatureView/FeatureOverview/StaleDialog/StaleDialog.tsx

```diff
@@ -30,12 +33,26 @@ const StaleDialog = ({ open, setOpen, stale }: IStaleDialogProps) => {
 
     const toggleActionText = stale ? 'active' : 'stale';
 
-    const onSubmit = async e => {
-        e.stopPropagation();
+    const onSubmit = async (event: React.SyntheticEvent) => {
+        event.stopPropagation();
         const patch = [{ op: 'replace', path: '/stale', value: !stale }];
         await patchFeatureToggle(projectId, featureId, patch);
```

Comment:
Hmm, this wasn't done in this PR. But it seems to be an oversight that this is not wrapped in a try catch?